apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -51,6 +51,19 @@ $tokensTable-border-width: 1px;
     }
 }
 
+.c-tokensTable-example--spacing {
+    width: var(--example-spacing);
+    height: var(--example-spacing);
+    background-color: #d7d9da;
+    display: block;
+    margin: 10%;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Could probably do `margin: 10% auto;` not sure it should be a `%` or a px value though

Reply:
ooh actually you might be able to do 
```
  align-self: center;
  justify-self: center;
```

Reply:
I'm not quite sure what this is for but I'm unsure whether we have any reason to use a percentage based margin - what was the need for this?

Reply:
ah you can ignore this 😄 updated to just use flexbox after I checked figma 👍